cmd/query/app/apiv3/otlp_translator.go

```diff
@@ -23,15 +23,26 @@ import (
 	"strconv"
 	"strings"
 
-	"go.opentelemetry.io/collector/translator/conventions"
-	tracetranslator "go.opentelemetry.io/collector/translator/trace"
+	"go.opentelemetry.io/collector/model/pdata"
+	semconv "go.opentelemetry.io/collector/model/semconv/v1.5.0"
 
 	"github.com/jaegertracing/jaeger/model"
 	commonv1 "github.com/jaegertracing/jaeger/proto-gen/otel/common/v1"
 	resourcev1 "github.com/jaegertracing/jaeger/proto-gen/otel/resource/v1"
 	tracev1 "github.com/jaegertracing/jaeger/proto-gen/otel/trace/v1"
 )
 
+const (
```

Comment:
These constants are now removed from https://github.com/open-telemetry/opentelemetry-collector/blob/v0.28.0/translator/trace/protospan_translation.go 

and moved to internal package https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/internal/coreinternal/tracetranslator/protospan_translation.go